### PR TITLE
fftw: version bump

### PR DESCRIPTION
**Pull request by junghans**

### CODE REVIEW

**Comment by s-sajid-ali** [comment]
FFTW@3.3.7 doesn't install properly with gcc@8.1.0[[1](https://github.com/FFTW/fftw3/commit/902d0982522cdf6f0acd60f01f59203824e8e6f3)].

**Comment by junghans** [comment]
@s-sajid-ali so should we add a conflict?

**Comment by s-sajid-ali** [comment]
It's good to do so to make sure no one accidentally ends up with a broken installation. @adamjstewart  ?

File: var/spack/repos/builtin/packages/fftw/package.py [hunk 1]
```diff
@@ -87,6 +88,8 @@ class Fftw(AutotoolsPackage):
     depends_on('automake', type='build', when='+pfft_patches')
     depends_on('autoconf', type='build', when='+pfft_patches')
     depends_on('libtool', type='build', when='+pfft_patches')
+    # https://github.com/FFTW/fftw3/commit/902d0982522cdf6f0acd60f01f59203824e8e6f3
+    conclits('%gcc@8:8.9999', when="@3.3.7")
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/fftw/package.py]:
```
==> Error: name 'conclits' is not defined
```

**Pull request merged**